### ISSUE TITLE
Authenticate to open.spotify.com using access token

### ISSUE THREAD

**Issue by fotiDim** [spotify/web-api#1457]
Is there any way to automatically authenticate a user to the [https://open.spotify.com](https://open.spotify.com) website when I valid his access and refresh token?

I noticed that a `sp_dc` cookie is all I need but I don't know how this relates to the access token and how I can generate one. Any hints would be appreciated.

Is there some other authentication mechanism that I don't know of?

**Comment by jscholes** [comment]
No.  The web player requires a token with scopes which are not valid for third party applications.  It would also violate user expectations, because when they authorise or revoke permissions for your app, they don't expect that action to also relate to open.spotify.com.

**Comment by fotiDim** [comment]
@jscholes let's assume that I have that token with the extra scopes. Is there a way to make the user be logged in the browser?
